app/src/pages/inside/logsPage/defectEditor/itemHeader/itemHeader.scss

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.item-info {
+  display: flex;
+  flex-direction: column;
+  padding: 8px 20px;
+  border-radius: 4px;
+  border: 2px solid transparent;
+  background-color: $COLOR--black-1;
+  cursor: pointer;
+  box-sizing: border-box;
+
+  &:hover:not(.selected) {
+    border-color: rgba($COLOR--topaz, 0.25);
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 18px;
+  margin-bottom: 12px;
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  color: $COLOR--topaz;
+  font-family: $FONT-REGULAR;
+  font-size: 13px;
+  text-decoration: none;
+
+  &:hover {
+    .icon {
+      opacity: 1;
+    }
+  }
+}
+
+.icon {
+  width: 15px;
+  height: 15px;
+  opacity: 0;
+  padding-left: 8px;
+
+  svg g g path {
+    fill: $COLOR--topaz;
+  }
+}
+
+.defect-type-override {
+  border: none;
+  background-color: transparent;
+  font-family: $FONT-REGULAR;
+  color: $COLOR--gray-light;
+  cursor: default;
+}
+
+.selected {
+  border: 2px solid $COLOR--topaz;
+}
+
+.bts-row {
+  min-height: 22px;
+}
+
+.issue-style-override {
```

Comment:
Do not specify that you override some styles - it is clear, because you are passing this class to another component.
Also not need to add `style` to the class name.